server/api/api/endpoints/pipelines.py

```diff
@@ -108,15 +108,12 @@ async def list_pipelines(
 async def create_pipeline(
     project: str,
     request: Request,
-    namespace: str = None,
     experiment_name: str = Query("Default", alias="experiment"),
     run_name: str = Query("", alias="run"),
     auth_info: mlrun.common.schemas.AuthInfo = Depends(deps.authenticate_request),
 ):
-    if namespace is None:
-        namespace = config.namespace
     response = await _create_pipeline(
-        auth_info, request, namespace, experiment_name, run_name, project
+        auth_info, request, experiment_name, run_name, project
```

Comment:
call them with kw args just to make sure
```suggestion
        auth_info=auth_info, 
        request=request,
        ...
```